std/process.d

```diff
@@ -132,6 +132,7 @@ version (Posix)
 {
     version (OSX)
     {
+        import std.internal.phobosinit; // needed to make sure the function gets called
```

Comment:
Please don't reference the encoding scheme init from std.process, merging everything in phobosinit is a really bad idea.